python/lsst/ts/m2gui/controltab/tab_utility_view.py

```diff
@@ -103,23 +103,23 @@ def _create_indicators_breaker(self):
 
         return indicators
 
-    def _update_indicator_color(self, indicator, is_fault):
+    def _update_indicator_color(self, indicator, is_triggered):
         """Update the color of indicator.
 
         Parameters
         ----------
         indicator : `PySide2.QtWidgets.QPushButton`
             Indicator.
-        is_fault : `bool`
-            Is in fault or not.
+        is_triggered : `bool`
```

Comment:
Picky comment, but maybe just call this "triggered"? Since this is a parameter should probably not start with a "verb"

Reply:
Done in the commit: [4ba6f54](https://github.com/lsst-ts/ts_m2gui/pull/25/commits/4ba6f544f1f07b33fe6074df114a7107e9287571). Thanks!